iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/database/project/DatabaseProjectXmlHandlerTest.java

```diff
@@ -0,0 +1,136 @@
+package eu.dnetlib.iis.wf.importer.database.project;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+
+import eu.dnetlib.iis.importer.schemas.Project;
+import eu.dnetlib.iis.wf.importer.RecordReceiver;
+
+public class DatabaseProjectXmlHandlerTest {
+
+    private String projectId = "40|arc_________::884b76698f227ecf30a1ace12071189a";
+    private String projectGrantId = "ANZCCART";
+    private String projectAcronym = "ANZ";
+    private String fundingClass = "ARC::ARC Centres of Excellences";
+    private String optional1 = "option1";
+    private String optional2 = "option2";
+    
+    private String rootTestResourcesPath = "/eu/dnetlib/iis/wf/importer/database/project/";
+    
+    private InputStream inputStream;
+    
+    private SAXParser saxParser;
+    
+    private ProjectsReceiver projectsReceiver;
+    
+    private DatabaseProjectXmlHandler handler;
+    
+    
+    //------------------------ TESTS --------------------------
+    
+    @Before
+    public void initialize() throws ParserConfigurationException, SAXException {
+        saxParser = SAXParserFactory.newInstance().newSAXParser();
+        handler = new DatabaseProjectXmlHandler(projectsReceiver = new ProjectsReceiver());
+    }
+    
+    @After
+    public void finalize() throws IOException {
+        if (inputStream!=null) {
+            inputStream.close();
+        }
+    }
+    
+    @Test
+    public void testProjectImportWithOptional() throws Exception {
+        // given
+        String filePath = rootTestResourcesPath + "project_with_optional.xml";
+        
+        // execute        
+        saxParser.parse(inputStream = DatabaseProjectXmlHandlerTest.class.getResourceAsStream(filePath), handler);
+        
+        // assert
+        assertEquals(1, projectsReceiver.getProjects().size());
+        Project project = projectsReceiver.getProjects().iterator().next();
```

Comment:
`.getProjects()` returns list, so why we don't simply use `.get(0)` instead of iterator?